src/librustc_resolve/error_reporting.rs

```diff
@@ -293,13 +293,20 @@ impl<'a> Resolver<'a> {
             (Def::Enum(..), PathSource::TupleStruct)
                 | (Def::Enum(..), PathSource::Expr(..))  => {
                 if let Some(variants) = self.collect_enum_variants(def) {
-                    err.note(&format!("did you mean to use one \
-                                       of the following variants?\n{}",
-                        variants.iter()
-                            .map(|suggestion| path_names_to_string(suggestion))
-                            .map(|suggestion| format!("- `{}`", suggestion))
-                            .collect::<Vec<_>>()
-                            .join("\n")));
+                    if !variants.is_empty() {
+                        let msg = if variants.len() == 1 {
+                            "use the enum's variant"
```

Comment:
```suggestion
                            "try using the enum's variant"
```
(For consistency with existing suggestions.)